### ISSUE TITLE
Min Temp Thresholds tool: Projected change time series labels are mis-positioned

### ISSUE THREAD

**Issue by daviswx** [ncsco/pinemap-dss-help#104]
This could be because the values are negative, and that negative value is carrying over into the label padding/position.  Needs more investigation.

**Comment by daviswx** [comment]
After investigating this, I determined that the label overlap with the columns is indeed occurring because of the negative values.  Highcharts sets the `verticalAlign` field to 'top' or 'bottom' based on the magnitude of the value.

I tried several different possible workarounds to try to override this, but they all gave errors or otherwise didn't work.  Finally, I manually set the `verticalAlign` field to 'middle', which puts the labels right at the top and bottom of the columns.  In my mind, this isn't ideal since it doesn't put any space between the edges of the columns and the labels, but it may have to suffice unless I spend another half day or more exploring further options.

![image](https://cloud.githubusercontent.com/assets/9257580/19120694/bc22df0a-8af1-11e6-9655-b0584e94857d.png)
